core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -861,8 +859,8 @@ private MaterializedResultWithPlan executeInternal(Session session, @Language("S
             verify(builder.get() != null, "Output operator was not created");
             return new MaterializedResultWithPlan(session.getQueryId(), builder.get().build(), plan);
         }
-        catch (IOException e) {
-            throw new UncheckedIOException(e);
+        catch (Exception e) {
+            throw new QueryFailedException(session.getQueryId(), e.getMessage(), e);
```

Comment:
this causes bunch of test failures.

Reply:
I figured that might be a problem.  I'll dig in more.